integrations/mssql.ini.tmpl

```diff
@@ -106,3 +106,10 @@ PATH = integrations/gitea-integration-mssql/data/lfs
 
 [packages]
 ENABLED = true
+
+[markup.html]
+ENABLED = false
+FILE_EXTENSIONS = .html
+RENDER_COMMAND = `go run tools/external_renderer.go`
+IS_INPUT_FILE = false
+RENDER_CONTENT_MODE=sanitized
```

Comment:
I think that is the wrong place to put this configuration...
The renderer should not be database dependent, right?
So why test it `x` times instead of once?
The database dependent tests are already the longest thing in our CI, I'd like to keep that as short as possible.